src/Core/Helpers/AbstractConfigHelper.php

```diff
@@ -33,16 +33,22 @@ abstract class AbstractConfigHelper {
   const INVENTORY_ITEMS_PER_PAGE = 500;
 
   /**
+   * Retrieve the configured client ID for connections to Wayfair's secure APIs
+   *
    * @return mixed
    */
   abstract public function getClientId();
 
   /**
+   * Retrieve the configured client secret for connections to Wayfair's secure APIs
+   *
    * @return mixed
    */
   abstract public function getClientSecret();
 
   /**
+   * Returns an Order Referrer Key
```

Comment:
The method name contains "ReferrerValue" and seems to imply that there is only one.
Is it actually returning an arbitrary Key, and not a value?

Reply:
yea that should say value not key